src/main/scala/scala/concurrent/stm/ccstm/Stats.scala

```diff
@@ -49,7 +49,7 @@ private[ccstm] object Stats {
 
     def contents: Seq[Long] = {
       val snap = _buckets map { _.apply() }
-      snap.take(1 + snap.lastIndexWhere { _ != 0L })
+      snap.toSeq.take(1 + snap.lastIndexWhere { _ != 0L })
```

Comment:
Why can’t `contents` return a `collection.Seq` instead, so that we don’t need the `.toSeq` call?